README.md

```diff
@@ -176,7 +176,7 @@ spec:
 
 ### Definition
 
-`CSI_LVM_DEVICE_PATTERN` is a grok pattern to specify which block devices to use for lvm devices on the node. This can be for example `/dev/sd[bcde]` if you want to use only /dev/sdb - /dev/sde.
+`CSI_LVM_DEVICE_PATTERN` is a grok pattern to specify which block devices to use for lvm devices on the node. This can be for example `/dev/sd[bcde]` if you want to use only /dev/sdb - /dev/sde. IMPORTANT: no wildcard (*) allowed actually.
 
 ### PVC Striped, Mirrored
```

Comment:
Please add a section "Local Development" or file DEVELOPMENT.md with all that loop-device magic